syntax = "proto3";

// input image, width, height
message ImageRequest {
    string farm =1;
    string sector = 2;
    string camIdx = 3;
}

// output prediction

message ImageResponse {
    bytes imgByte1 = 4;
    bytes imgByte2 = 5;
}

// service
service ImageServer {
    rpc getImage(ImageRequest) returns (ImageResponse) {}
}